### PR TITLE
Taking & putting clothes from and on heads

### PR DESCRIPTION
**Pull request by BatElite**
[feat][qol]
## About the PR 
This PR lets players take stuff from decapitated heads, as well as put appropriate clothing types on them.

Also pokes the variable for the ear item, which could previously be a headset in practice even though those aren't of type ```obj/item/clothing/ears/```.

## Why's this needed? 
Not being able to access a dead person's stuff without reattaching their head to a torso first was unintuitive and needlessly complicated. It's mostly an issue when there's a katana rampage going on.
I can't imagine much practical use for dressing up a head, but who am I to limit the macabre interests of the crew?

## Changelog

```
(u)BatElite:
(*)You can now take stuff from loose heads. Or put things on them, if you're into head spike fashion shows.
```

### CODE REVIEW

File: code/obj/item/organs/head.dm [hunk 1]
```diff
@@ -258,7 +258,61 @@
 		. = ..()
 		src.transplanted = 1
 
+	///Taking items off a head
+	attack_self(mob/user as mob)
+		var/list/headwear = list(head, ears, wear_mask, glasses)
+		if (!length(headwear))
+			return
```
Comment (flrsh) [inline code/obj/item/organs/head.dm:265]: Pretty sure a list of `null` values still has a length.

File: code/obj/item/organs/head.dm [hunk 2]
```diff
@@ -258,7 +258,61 @@
...
+		var/obj/selection = input("Which item do you want to remove","Looting the dead") as null|obj in headwear
+		if (!selection)
+			return
+		user.put_in_hand_or_drop(selection)
+		//switch(selection)
```
Comment (flrsh) [inline code/obj/item/organs/head.dm:270]:
```suggestion
```

File: code/obj/item/organs/head.dm [hunk 3]
```diff
@@ -258,7 +258,61 @@
...
+				update_head_image()
+			else boutput(user, "<span class='alert'>[src.name] is already wearing [src.wear_mask.name]!</span>")
+		if (istype(W, /obj/item/clothing/glasses))
+			if (!glasses)
+				user.drop_item(W)
+				glasses = W
+				glasses.set_loc(src)
+				update_head_image()
+			else boutput(user, "<span class='alert'>[src.name] is already wearing [src.glasses.name]!</span>")
+
```
Comment (flrsh) [inline code/obj/item/organs/head.dm:314]: Should do an early return.

**Review by flrsh — CHANGES_REQUESTED** [review]

File: code/obj/item/organs/head.dm (hunk 1 above)
Comment (BatElite) [reply code/obj/item/organs/head.dm]: Indeed, and it turns out I didn't realise the ```if (!selection)``` already did what I was trying to do.

**Review by flrsh — APPROVED** [review]

**Pull request merged**